packages/safe-core-sdk/README.md

```diff
@@ -651,12 +651,12 @@ const txHash = await safeSdk.getTransactionHash(safeTransaction)
 const owners = await safeSdk.getOwnersWhoApprovedTx(txHash)
 ```
 
-### getEnableModuleTx
+### createEnableModuleTx
```

Comment:
Should we format these, i.e.

```suggestion
### `createEnableModuleTx`
```

Reply:
I'm personally not a fan of using `` in titles like these from the visuals perspective. I found ENS is doing it [this way](https://github.com/ensdomains/docs/blob/master/contract-api-reference/publicresolver.md), but also not sure which one is better. I think I prefer to keep the method names as they are, all together.

Reply:
Sure, let's keep them as is.